### PR TITLE
NB Datasets

### PR DESCRIPTION
**Pull request by Isaac-Flath**
@jph00 

This PR allows users to specify datasets to be attached to kernels when they create new kernels.  This is useful if they have datasets that contain model weights or libraries needed for offline install.

### CODE REVIEW

File: 00_core.ipynb [hunk 1]
```diff
@@ -148,7 +148,7 @@
    "outputs": [],
```
Comment (jph00) [inline 00_core.ipynb:148]: <p>Careful - don't use <code>[]</code> as a default! https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastkaggle/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

Comment (jph00) [inline 00_core.ipynb:148]: <p><code>[]</code> here too </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastkaggle/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

Comment (Isaac-Flath) [reply 00_core.ipynb:148]: <p>Good to know - Thanks!</p><p> </p><p>Fixed.</p>

**Comment by idanre1** [comment]
Is it possible to make a new fastkaggle release with that PR?
Thanks in advance !

**Comment by jph00** [comment]
Sure - just done. :)

**Pull request merged**